.circleci/config.yml

```
@@ -154,54 +170,94 @@ commands:
-              if command -v curl >/dev/null 2>&1; then
-                curl -L -o miniconda.sh "$MINIFORGE_URL"
-              else
-                echo "Error: no wget, curl, or python available to download Miniconda." >&2
-                exit 1
-              fi
+              echo "Error: no wget, curl, python3, or python available to download Miniconda." >&2
+              exit 1
             fi
 
+            # Verify SHA256 checksum to prevent supply chain attacks
+            # Resolve the final versioned URL (since 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)